src/illuminatio/rule.py

```diff
@@ -1,5 +1,8 @@
-import kubernetes as k8s
+"""
+file for the creation of networking rules
```

Comment:
Start with a capital letter to match the rest of the docstrings (see: https://www.python.org/dev/peps/pep-0257/)